charts/apisix/values.yaml

```diff
@@ -233,11 +234,13 @@ gateway:
     enabled: true
     servicePort: 80
     containerPort: 9080
+    nodePort: ""
```

Comment:
When using the empty string as the default value of `nodePort`, it might mislead users to configure a string for `nodePort`. `nodePort` should be an integer, let's just comment the `nodePort` when we don't need it.

Reply:
Changes have been made, comments have been added and resubmitted